web/src/utils/helpers.ts

```diff
@@ -108,3 +108,33 @@ export function getRenewableRatio(
 ): number {
   return (isConsumption ? renewableRatio : renewableRatioProduction) ?? Number.NaN;
 }
+
+/**
+ * Function to round a number to a specific amount of decimals.
+ * @param {number} number - The number to round.
+ * @param {number} decimals - Defaults to 2 decimals.
+ * @returns {number} Rounded number.
+ */
+export const round = (number: number, decimals = 2): number => {
```

Comment:
Hmm, not sure why we need this function compared to `.toFixed()`? We also have the same in `geo/utilities.ts`, dunno if we should try merging those or just leave it like this

Reply:
Technically to fixed don't round things but cuts them off.

So 5.967 would be 5.96 instead of 5.7 with toFixed (that is if I remember correctly).

But I'm hoping we move the netExchage calculation to the app-backend (netCO2exchange is already in there) and maybe rework how the formatting work so we return the correct number instead of the division factor.

But that's something for future improvements.

Reply:
Here is another difference between the two:
```

> const round = (number, decimals = 2) => {  return (    (Math.round((Math.abs(number) + Number.EPSILON) * 10 ** decimals) / 10 ** decimals) *    Math.sign(number)  );};
undefined
> round(0.0001)
0
> 0.0001.toFixed(2)
'0.00'
>
```
So I do like "my" round function better as it gives:

1. Number output
2. The expected amount of digits.

Reply:
Good point, thanks for the explanation - let's stick with yours:)